index.html

```diff
@@ -0,0 +1,62 @@
+<html>
+    <head>
+        <link href="css/style.css" rel="stylesheet">
+        <meta charset="utf-8">
+    </head>
+    <body>
+        <header>
+            <div class="wrapper">
+                <div class="block"></div>
+                <div class="block">
+                    Something text
+                </div>
+                <div class="block"></div>
+            </div>
+        </header>
+        <div class="content-wrapper">
```

Comment:
it seems `main` tag should be here, in additional, why not to use `.wrapper`, it has 1200